setup.py

```diff
@@ -153,7 +153,8 @@ def run(self):
         import django
         import codecs
         from pootle.apps.pootle_misc.checks import check_names, excluded_filters
-        from translate.filters.checks import TeeChecker
+        from translate.filters.checks import (
+            TeeChecker, StandardChecker, StandardUnitChecker)
```

Comment:
Please apply our styleguide here, http://docs.translatehouse.org/projects/pootle/en/latest/developers/styleguide.html#imports which extends http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#imports So it would be:

```python
        from translate.filters.checks import (StandardChecker,
                                              StandardUnitChecker, TeeChecker)
```